lib/sufia/export/converter.rb

```diff
@@ -0,0 +1,22 @@
+module Sufia
+  module Export
+    # a base class to convert an ActiveFedora object that contains permissions and allow for pretty json.
+    #
+    class Converter
+      # overrides to_json to optionally allow for a pretty version of the json to be outputted
+      #
+      # @param [Boolean] pretty pass true to output formatted json using pretty_generate
+      def to_json(pretty = false)
+        json = super()
+        return json unless pretty
+        JSON.pretty_generate(JSON.parse(json))
```

Comment:
@mjgiarlo Went with a third option suggested by @awead and approved as readable by @mtribone. 👌 ?